app/view_objects/assessor_interface/further_information_request_view_object.rb

```diff
@@ -27,6 +27,7 @@ def review_items
       .order(:created_at)
       .map do |item|
         {
+          further_information_request_item: item,
```

Comment:
I think we should be able to remove this from the items returned from this method.